apps/frontend/src/app/5_pages/AavePage/components/LendAssetsList/components/LendAssetAction/LendAssetAction.tsx

```diff
@@ -1,36 +1,25 @@
-import React, { FC, useState } from 'react';
+import React, { FC } from 'react';
 
 import { t } from 'i18next';
 import { useNavigate } from 'react-router-dom';
 
 import { Button, ButtonStyle } from '@sovryn/ui';
 
 import { translations } from '../../../../../../../locales/i18n';
-import { LendPoolDetails } from '../../LendAssetsList.types';
-import { LendModalContainer } from '../LendModal/LendModalContainer';
 
 type LendAssetActionProps = {
```

Comment:
move types and constants outside the component (types folder maybe :) )

Reply:
same, I would rather keep it their style here ))